security-force-monitor/sfm-cms
# Update public-facing templates for field harmonization

## Overview

Make sure that the new field names are reflected in all public-facing views. As part of this work, update all links to external documentation to make sure they point to the updated docs.

In addition, fix a small bug #564 that is blocking the SFM team from checking sources.

Connects #662, #656, #564.

## Testing Instructions

* In working on this PR, I noticed we'd configured the Solr service incorrectly in Docker, meaning that search indexes weren't being persisted when the service was taken down. This means your Solr index probably doesn't exist, so recreate it by running `docker-compose up` in one shell and `docker-compose run --rm app ./manage.py make_search_index --recreate` in another.
* For each of personnel, units, and incidents, do:
    * Check the search page and confirm that the filters and column names match and look correct
    * Click on a detail view and confirm that the table columns also look good
    * Click on all of the help links (the grey question marks) on the page and confirm that you get taken to the proper help documentation (in each case, you should be taken to a specific section of the help page)
* Confirm the fix for the bug in #564 by visiting any detail page and opening up a citation. You should be able to click links in the citation, like the help link and the links to sources, without closing the citation popover; you should also be able to click on the body of the page outside of the citation popover to close it automatically.
